components/Table.tsx

```diff
@@ -46,12 +46,12 @@ export const Table = ({columns, data}: TableProps ): JSX.Element =>  {
         {rows.map(row => {
           prepareRow(row)
           return (
-            <Tr>
-              {row.cells.map(cell => {
+            <Tr {...row.getRowProps}>
+              {row.cells.map(cell => (
                 <Td {...cell.getCellProps()}>
                   {cell.render("Cell")}
                 </Td>
-              })}
+              ))}
```

Comment:
Having these brackets instead of parentheses caused nothing to render because brackets indicate a function body but there was no return statement